Automattic/woocommerce-payments
# Open the payment method selector from the payment methods list

Fixes #1671

#### Changes proposed in this Pull Request

Opens the payment methods selector modal window when the `Add payment method` button is clicked. The modal window is passed the currently enabled payment methods, and they are preselected by default.

#### Testing instructions

1. Open the settings screen: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
2. Click the `Add payment method` button. The popup should appear.
3. Click the close button, the `Cancel` button or out of the modal window. The window should close.

-------------------
